web/impact/impact/utils.py

```diff
@@ -12,6 +13,15 @@
         "2010-01-01T00:00:00Z",
         "%Y-%m-%dT%H:%M:%SZ"))  # format based on browsable API output
 
+UPDATE_STATEMENT = "UPDATE {table} SET updated_at = %s WHERE id = %s"
+
+
+def override_updated_at(model, value):
+    # Yes, this big of stick is needed to change the update_at field
+    with connection.cursor() as cursor:
+        sql = UPDATE_STATEMENT.format(table=model._meta.db_table)
```

Comment:
interesting - as long as this is here only for tests. I think we should leave the actual field as readonly

Reply:
It pretty much is already.  They are marked as allowing None/NULL which they have to be due to the old data, but it's a pain in  the butt to get them to actually be any value other than what Django wants them to be so they are effectively read-only.